### PR TITLE
@damassi => [Test] Update FollowArtistPopover to use renderRelayTree

### PR DESCRIPTION
**Pull request by mzikherman**
Last test that was using `RelayStubProvider`! 

Up next, storybooks...

### CODE REVIEW

File: src/Components/v2/FollowArtistPopover/index.tsx [hunk 1]
```diff
@@ -77,8 +77,8 @@ const FollowArtistPopover: SFC<Props> = props => {
 export const FollowArtistPopoverFragmentContainer = createFragmentContainer(
   FollowArtistPopover,
   {
-    suggested: graphql`
-      fragment FollowArtistPopover_suggested on Artist {
+    artist: graphql`
```
Comment (mzikherman) [inline src/Components/v2/FollowArtistPopover/index.tsx]: cc @ds300 for some reason when this fragment container's prop was aliased, i could not get the `mockData` and `renderRelayTree` setup to work no matter how much I massaged the data.

Switching this back to the more 'default' `artist` (as in, what type the fragment spreads and what's returned by the query), and everything was fine.

Aliasing _within_ a fragment, like:

```
shows1: shows(fair: true) { ... }
shows2: shows(fair: false) { ... }
```

was working properly, it was this data prop alias that somehow wasn't able to work.

**Comment by artsyit** [comment]
:rocket: PR was released in v21.9.6 :rocket:

**Comment by mzikherman** [comment]
cc @alloy

File: src/Components/v2/FollowArtistPopover/index.tsx (hunk 1 above)
Comment (alloy) [reply src/Components/v2/FollowArtistPopover/index.tsx]: Interesting, I wonder is that’s because it’s a root field selection?

Comment (ds300) [reply src/Components/v2/FollowArtistPopover/index.tsx]: oh I thought I had replied to this last week. must have forgotten to hit send.

anyway, yes, I think it doesn't support root field aliases right now. Can't remember why but would be worth looking into again.

Comment (alloy) [reply src/Components/v2/FollowArtistPopover/index.tsx]: Great opportunity for the two of you to pair on.

**Pull request merged**